changes/4143-homeworkprod.md

```diff
@@ -0,0 +1 @@
+Tweak documentation: Fix command to install python-dotenv, align spelling
```

Comment:
Do we really need to mention this in changes?

Reply:
I'm fine with not having this in the changelog.

I just added the file because the PR template requested me to.

Reply:
I think we can remove this. please update the PR. Thanks!

Reply:
Done.